consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/util/McolDefenceDetailTypeUtilTest.java

```diff
@@ -0,0 +1,161 @@
+package uk.gov.moj.sdt.consumers.util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.moj.sdt.cmc.consumers.model.McolDefenceDetailType;
+import uk.gov.moj.sdt.cmc.consumers.model.McolDefenceDetailTypes;
+import uk.gov.moj.sdt.cmc.consumers.model.claimdefences.ClaimDefencesResult;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Test class for the consumer gateway.
+ *
+ * @author Mark Dathorne
+ */
+@ExtendWith(MockitoExtension.class)
+class McolDefenceDetailTypeUtilTest {
+
+    private ClaimDefencesResultsUtil claimDefencesResultsUtil;
+    private McolDefenceDetailTypeUtil mcolDefenceDetailTypeUtil;
+    /**
+     * Method to do any pre-test set-up.
+     */
+    @BeforeEach
+    public void setUpLocalTests() {
+        claimDefencesResultsUtil = new ClaimDefencesResultsUtil();
+        mcolDefenceDetailTypeUtil = new McolDefenceDetailTypeUtil();
+    }
+
+    @Test
+    void shouldConvertMcolDefenceDetailTypeJAXBToXml() {
+
+        McolDefenceDetailType detailType = mcolDefenceDetailTypeUtil.convertToMcolDefenceDetailType(
+                claimDefencesResultsUtil.createClaimDefencesResult()
+        );
+
+        JAXBElement<McolDefenceDetailType> jaxbDetailType = mcolDefenceDetailTypeUtil.createMcolDefenceDetail(detailType);
+        StringWriter stringWriter = new StringWriter();
+        try {
+            JAXBContext jaxbContext = JAXBContext.newInstance(McolDefenceDetailType.class);
+            Marshaller marshaller =  jaxbContext.createMarshaller();
+            marshaller.marshal(jaxbDetailType, stringWriter);
+        } catch (JAXBException e) {
+            throw new RuntimeException(e);
```

Comment:
Throwing a new RuntimeException might be flagged as an issue by SonarQube.  Try changing it to fail(e.getMessage()).